### PR TITLE
[AIRFLOW-XXX] Updated readme with "Who's using" information

### PR DESCRIPTION
**Pull request by saianupkumarp**
We are using Airflow at King Abdullah Petroleum Studies and Research Center(KAPSARC)(www.kapsarc.org)

### CODE REVIEW

**Comment by feng-tao** [comment]
need rebase

**Comment by saianupkumarp** [comment]
> need rebase

Thank you @feng-tao. I have done the rebase.

File: README.md [hunk 1]
```diff
@@ -282,6 +282,7 @@ Currently **officially** using Airflow:
 1. [JobTeaser](https://www.jobteaser.com) [[@stefani75](https://github.com/stefani75) &  [@knil-sama](https://github.com/knil-sama)]
 1. [JULO](https://www.julo.co.id/) [[@sepam](https://github.com/sepam) & [@tenapril](https://github.com/tenapril) & [@verzqy](https://github.com/verzqy)]
 1. [Kalibrr](https://www.kalibrr.com/) [[@charlesverdad](https://github.com/charlesverdad)]
+1. [King Abdullah Petroleum Studies and Research Center(KAPSARC)](https://github.com/kapsarc) [[@saianupkumarp](https://github.com/saianupkumarp)]
```
Comment (feng-tao) [inline README.md]: could you to make it alphabetically? Thanks a lot.

**Review by feng-tao — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (saianupkumarp) [reply README.md]: My bad, I should've done this earlier. Corrected now. Thank you!

**Pull request merged**